api/envoy/data/tap/v3/http.proto

```diff
@@ -34,6 +36,9 @@ message HttpBufferedTrace {
 
     // Message trailers.
     repeated config.core.v3.HeaderValue trailers = 3;
+
+    // Message caught time stamp.
+    google.protobuf.Timestamp caught_time = 4;
```

Comment:
Hi @adisuissa ,
This isn't the first byte timestamp. As you know, in our HTTP Tap filter, once onRequest header is called, all Headers in HTTP request is received and decoded. So from HTTP pointer of view, this could be request caught time.
thanks
Cliff

Reply:
Per other comment let's rename and clarify the comment.

Reply:
sure. will do in above comment